modules/animation/include/modules/animation/mainanimation.h

```diff
@@ -88,9 +88,10 @@ class IVW_MODULE_ANIMATION_API MainAnimation {
      */
     void addKeyframeSequenceCallback(Property* property);
 
-    // create this lazy so we have all the factories ready when we create it
     Animation* animation_;
     InviwoApplication* app_;
+    
+    // create this lazy so we have all the factories ready when we create it
```

Comment:
I think that this comment applies to animation_ as well

Reply:
we don't create the animation we are given it in the constructor...